src/app/overview/widgets/stakinginfo/stakinginfo.component.html

```diff
@@ -1,36 +1,22 @@
-<div class="staking-info">
-  <app-header>
-    <ul class="header-nav">
-      <li>Staking Information</li>
-    </ul>
-    <!--ul class="header-nav pull-right">
-      <li><a href="javascript:void(0)"><span class="icon fa fa-cog color-green"></span></a></li>
-    </ul-->
-  </app-header>
-  <md-card>
-    <table class="staking-overview">
-      <tr>
-        <th>Next reward</th>
-        <td>~3 hours</td>
-      </tr>
-      <tr>
-        <th>Your Stake weight</th>
-        <!-- (your weight) / (network weight) * 1000 -->
-        <td>3.<small>48</small>&nbsp;&permil;</td>
-      </tr>
-      <tr>
-        <th>Stake percentage</th>
-        <td>{{splitAmountByDot(dynamicStakingReward)}}.<small>{{splitAmountByDot(dynamicStakingReward,true)}}</small>&nbsp;%</td>
-      </tr>
-      <tr>
-        <th>Stake reward</th>
-        <td>1.<small>4949327</small>&nbsp;PART</td>
-      </tr>
-    </table>
-    <!-- .staking-overview -->
-  </md-card>
-</div>
-<!-- .staking-info -->
```

Comment:
This code ^ is AFAIK the right one. Seems like bad merge or something..

Reply:
Looks like bad merge yes

Reply:
not sure, As its saying up to date to my branch lol :D